runtime/src/tests/functionalTests/page_objects/app.page.ts

```diff
@@ -0,0 +1,37 @@
+import { browser, ExpectedConditions as until, $, by } from 'protractor';
+import * as support from '../support';
+import { BaseElement } from '../ui';
+
+import { BasePage } from './base.page';
+
+export abstract class AppPage extends BasePage {
+  appTag = $('f8-app');
+
+  /**
+   * Extend this class, to describe Application Page(after logging in)
+   *
+   * @param {url} string URL where the extended page resides
+   */
+  constructor(url?: string) {
+    super(url);
+  }
+
+  /**
+   * Returns an instance of the BaseElement that can be found using
+   * the {css} and contains the {text}.
+   *
+   * @param {UI} The Base Element Class e.g. Button, TextInput
```

Comment:
I see no reason to uppercase it. Just `ui` should be fine.

Reply:
Done.